### ISSUE TITLE
path for screenshot shortcuts is en_US only

### ISSUE THREAD

**Issue by killajoe** [EndeavourOS-Community-Editions/bspwm#23]
https://github.com/EndeavourOS-Community-Editions/bspwm/blob/main/.config/sxhkd/sxhkdrc
```
# Screenshot
Print
  scrot ~/Pictures/Screenshot-$(date +%F_%T).png

super + Print
  ~/.config/bspwm/scripts/screenshot.sh
```
https://github.com/EndeavourOS-Community-Editions/bspwm/blob/main/.config/bspwm/scripts/screenshot.sh
```
        cd ~/Pictures/ && sleep 1 && scrot;;
    $option1)
        cd ~/Pictures/ && scrot -s;;
    $option2)
        cd ~/Pictures/ && sleep 1 && scrot -u;;
```

I am sure it will not work if you use a different language where the folder will get named in the language chosen.. 

`XDG_PICTURES_DIR="$HOME/Pictures"`
in
`~/.config/user-dirs.dirs`
it could take it from there or is there something like $HOME also for the other folders? or it could get defined somewhere..

**Comment by flyingcakes85** [comment]
@killajoe 

Could you please see #24 and check if its ok

Thanks

**Comment by flyingcakes85** [comment]
fixed in #25
